README.md

```diff
@@ -1,5 +1,47 @@
+![Netlify examples](netlify-badge-examples.png)
+
 # Netlify Feature Tour
 
-> **NOTE:** This is a work in progress.
+**Access this demo site**: https://feature-tour.netlify.app
+
+[![Netlify Status](https://api.netlify.com/api/v1/badges/fad6792e-1c44-44db-bd79-ea74b42b0f89/deploy-status)](https://app.netlify.com/sites/feature-tour/deploys)
+
+## About this example site
+
+This site provides a path to get started learning about Netlify features.
+
+- 📚 [Docs Getting Started Tutorial](https://docs.netlify.com/get-started/?utm_medium=social&utm_source=github&utm_campaign=devex-ph&utm_content=devex-examples)
+
+## Speedily deploy your own version
+
+Deploy your own version of this example site by selecting the Deploy to Netlify Button below. This will automatically:
+
+- Clone a copy of this repo to your own GitHub account
+- Create a new project in your [Netlify account](https://app.netlify.com/?utm_medium=social&utm_source=github&utm_campaign=devex-ph&utm_content=devex-examples), linked to your new repo
```

Comment:
I don't think I can take credit for the traffic on this link. Probably just a copy/paste hangover

```suggestion
- Create a new project in your [Netlify account](https://app.netlify.com/?utm_medium=social&utm_source=github&utm_campaign=devex&utm_content=devex-examples), linked to your new repo
```